### PR TITLE
Update README.md to include ternary conditionals styling

### PR DESCRIPTION
**Pull request by robwise**
[Review on Reviewable](https://reviewable.io/reviews/shakacode/style-guide-javascript/5)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,3 +104,49 @@ Here is why:
...
+  return() {
+    return (
+      {
+        someBoolean ?
+        <SomeComponent /> :
+        <OtherComponent />
+      }
+    );
+  }
+}
```
Comment (alex35mil) [inline README.md:151]: This is strange part. It looks like broken example from line `113`.

File: README.md [hunk 2]
```diff
@@ -104,3 +104,49 @@ Here is why:
...
+);
+```
+
+The same can be applied to assignments:
+
+```
+return ( // <-- Invitation to see the expression on the next line. Result of the expression will be returned.
+ ... //  <-- Expression encapsulated here and looks solid, easier to read
+);
+```
```
Comment (alex35mil) [inline README.md:136]: I'd use here actual assignment:

``` js
const myConst = ( // <---
  ... // <---
);
```

**Comment by alex35mil** [comment]
@robwise Looks good! Few comments.

**Comment by robwise** [comment]
@alexfedoseev sorry for the delay, fixed those issues

**Comment by justin808** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

_[README.md, line 143 [r2]](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5#-K95asfDIRQ5NO7URlb_:-K95asfEGoskga40VIgs:1427514053) ([raw file](https://github.com/shakacode/style-guide-javascript/blob/618a0a28ef4492a29bc4c2de5e72283593c5f48e/README.md#L143)):_
this doesn't look right

``` js
return (
      someBoolean ?
      <SomeComponent /> :
      <OtherComponent />
     );
```

braces are wrong

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5)_

**Comment by robwise** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[README.md, line 143 [r2]](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5#-K95asfDIRQ5NO7URlb_:-K95c93-lvK3Pqx6d2rg:676108759) ([raw file](https://github.com/shakacode/style-guide-javascript/blob/618a0a28ef4492a29bc4c2de5e72283593c5f48e/README.md#L143)):_
@justin808 how so ?

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5)_

**Comment by alex35mil** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[README.md, line 143 [r2]](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5#-K95asfDIRQ5NO7URlb_:-K96pXLhDzCoCKXOVzQB:-1052474637) ([raw file](https://github.com/shakacode/style-guide-javascript/blob/618a0a28ef4492a29bc4c2de5e72283593c5f48e/README.md#L143)):_
Yep, it's not valid JSX, b/c there is no root React element. This one will be valid:

``` js
render() {
  return (
    <div>
      {
        someBoolean ?
        <SomeComponent /> :
        <OtherComponent />
      }
    </div>
  );
}
```

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5)_

**Comment by alex35mil** [comment]
One comment and my :+1: to merge!

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5#-:-K96qHv0qx4JuwYIwOyP:-379761709)_

**Comment by robwise** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[README.md, line 143 [r2]](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5#-K95asfDIRQ5NO7URlb_:-K98XUkrSnxe1_nCH6p6:-499146009) ([raw file](https://github.com/shakacode/style-guide-javascript/blob/618a0a28ef4492a29bc4c2de5e72283593c5f48e/README.md#L143)):_
Done.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/5)_

**Pull request merged**